### PR TITLE
edge cases  of the withdraw method resolved

### PR DESCRIPTION
**Pull request by rchanRakshith**

### CODE REVIEW

**Review by Manish-EG — APPROVED** [review]
It is perfect

File: BankManagementSystem/Program.cs [hunk 1]
```diff
@@ -140,7 +140,12 @@ static void Main(string[] args)
                          password = Console.ReadLine();
                         Console.WriteLine("Enter the amount to withdraw");
                         amount = Convert.ToDouble(Console.ReadLine());
-                        accountControllerObj.Withdraw(accountNumber, password, amount);
+                        if (customerObject.AccountDetails.Balance <= amount)
+                            accountControllerObj.Withdraw(accountNumber, password, amount);
+                        else
+                        {
+                            Console.WriteLine($"Please enter the amount less than  the balance amount {customerObject.AccountDetails.Balance}");
```
Comment (sheetal2605) [inline BankManagementSystem/Program.cs]: Add one condition to check for negative values

Comment (rchanRakshith) [reply BankManagementSystem/Program.cs]: please check it now

**Review by sheetal2605 — APPROVED** [review]
Perfect

**Pull request merged**